formlibrary/models/case.py

```diff
@@ -15,37 +20,91 @@ class Case(models.Model):
+    secondary_phone = models.CharField(validators=[phone_regex], max_length=17, blank=True)
+    email = models.CharField(validators=[email_regex], max_length=100, blank=True)
 
 
-class Individual(CreatedModifiedDates):
+IMAGE_SPEC = {
+    "width": 2000,
+    "height": 500,
+    "limit_kb": 100
+}
+
+
+def validate_image(image, width=IMAGE_SPEC['width'],
+                   height=IMAGE_SPEC['height'],
+                   limit_kb=IMAGE_SPEC['limit_kb']):
+    file_size = image.file.size
+    if file_size > limit_kb * 1024:
+        raise ValidationError("Max size of file is %s KB" % limit_kb)
+    w, h = get_image_dimensions(image)
+    if w < width:
+        raise ValidationError("Min width is %s" % width)
+    if h < height:
+        raise ValidationError("Min height is %s" % height)
+
+
+class Individual(Case, CreatedModifiedDates, CreatedModifiedBy):
     """
     Individual, or person.
     Subject to future changes: https://github.com/hikaya-io/activity/issues/403
     Also, will inherit from Case (subject to research/discussion)
     """
+    SEX_CHOICES = (
+        ('M', 'Male'),
+        ('F', 'Female')
+    )
     first_name = models.CharField(max_length=255, null=True, blank=True)
-    training = models.ManyToManyField("formlibrary.training", blank=True)
-    distribution = models.ManyToManyField("formlibrary.distribution", blank=True)
-    father_name = models.CharField(max_length=255, null=True, blank=True)
-    age = models.IntegerField(null=True, blank=True)
-    gender = models.CharField(max_length=255, null=True, blank=True)
-    site = models.ForeignKey(SiteProfile, null=True,
-                             blank=True, on_delete=models.SET_NULL)
+    last_name = models.CharField(max_length=255, null=True, blank=True)
+    sex = models.CharField(choices=SEX_CHOICES, max_length=1, null=True, blank=True)
+    date_of_birth = models.DateField(null=True, blank=True)
+    household_id = models.ForeignKey(
+        Household, null=True, blank=True, on_delete=models.SET_NULL)
+    head_of_household = models.BooleanField(default=True)
+    id_type = models.CharField(max_length=255, null=True, blank=True)
+    id_number = models.CharField(max_length=255, null=True, blank=True)
+    phone_regex = RegexValidator(
+        regex=r'^\+?1?\d{9,15}$', message="Invalid Phone Number. Format: '+123456789'. Up to 15 digits allowed.")
+    primary_phone = models.CharField(validators=[phone_regex], max_length=17, blank=True)
+    secondary_phone = models.CharField(validators=[phone_regex], max_length=17, blank=True)
     signature = models.BooleanField(default=True)
-    remarks = models.TextField(max_length=550, null=True, blank=True)
-    program = models.ManyToManyField(Program, blank=True)
+    site = models.ForeignKey(
+        SiteProfile, null=True, blank=True, on_delete=models.SET_NULL)
+    photo = models.ImageField(upload_to="media/images", validators=[validate_image], blank=True, null=True)
```

Comment:
Please also rename the files otherwise this might end up overriding exiting images with same names.